app/models/invenio_rdm_record_converter.rb

```diff
@@ -46,6 +49,11 @@ def initialize(generic_file=nil)
     # @permissions = permissions(generic_file)
   end
 
+  def to_json(options={})
```

Comment:
As I mentioned before, this is necessary to pass the options hash which in turn makes it so we can exclude certain variables from the object when we convert it to JSON format.